aws/ecs_cluster_test.go

```diff
@@ -0,0 +1,130 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test that we can succesfully list ECS clusters by manually creating a cluster and then using the list function to find it.
+func TestCanCreateAndListEcsCluster(t *testing.T) {
+	t.Parallel()
+
+	region := "eu-west-1"
+	awsSession, err := session.NewSession(&awsgo.Config{
+		Region: awsgo.String(region),
+	})
+	if err != nil {
+		assert.Fail(t, errors.WithStackTrace(err).Error())
+	}
+
+	clusterName := "test-ina-1"
```

Comment:
Since we run tests in parallel—both within a single test suite running on one computer, and many tests in various branches running in CircleCi—we need to ensure that names are unique. We have a convenient [`UniqueId`](https://github.com/gruntwork-io/cloud-nuke/blob/master/util/unique_id.go#L12) method for this:

```suggestion
	clusterName := fmt.Sprintf("test-ecs-cluster-%s", util.UniqueId())
```